bazel/linux/templates/runner.aarch64_template.sh

```diff
@@ -0,0 +1,170 @@
+#!/bin/bash
```

Comment:
same thing here -- can we parameterize this script and use it for both x86 and aarch64?

Reply:
Ok did that now.